### ISSUE TITLE
Training set and validation set division

### ISSUE THREAD

**Issue by 212wzt5A** [motional/nuplan-devkit#45]
Hi Nuplan team,
    I saw that in the yaml file of the splitter, the log names of the training set, validation set, and test set are specified. I would like to ask how this data set is divided, such as by different areas of the city, or by time.

**Comment by mh0797** [comment]
Additionally, the dataset page does not mention in which folder the test set belongs. 
I assume it belongs in the trainval folder as well since there is a dedicated splitter to handle the different splits. Nonetheless an explicit hint on the dataset readme would be useful.
